test/functionalTest/cases/1875_NGSIv2_notification_formats/NGSIv2_notification_formats.test

```diff
@@ -0,0 +1,509 @@
+                        "type": "none", 
+                        "value": 1
+                    }, 
+                    {
+                        "name": "A2", 
+                        "type": "none", 
+                        "value": 2
+                    }
+                ], 
+                "id": "E1", 
+                "isPattern": "false", 
+                "type": "none"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
+        }
+    ], 
+    "originator": "localhost",
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 170
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: localhost:REGEX(\d+)
+Accept: application/json
+X-Ngsiv2-Attrsformat: normalized
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A1": {
+                "metadata": {}, 
+                "type": "none", 
+                "value": 1
+            }, 
+            "A3": {
+                "metadata": {}, 
+                "type": "none", 
+                "value": 3
+            }, 
+            "id": "E1", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 170
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: localhost:REGEX(\d+)
+Accept: application/json
+X-Ngsiv2-Attrsformat: normalized
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A1": {
+                "metadata": {}, 
+                "type": "none", 
+                "value": 1
+            }, 
+            "A4": {
+                "metadata": {}, 
+                "type": "none", 
+                "value": 4
+            }, 
+            "id": "E1", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 94
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: localhost:REGEX(\d+)
+Accept: application/json
+X-Ngsiv2-Attrsformat: keyvalues
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
+        {
+            "A1": 1, 
+            "A5": 5, 
+            "id": "E1", 
+            "type": "none"
+        }
+    ], 
+    "subscriptionId": "REGEX([0-9a-f]{24})"
+}
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Content-Length: 60
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Host: localhost:REGEX(\d+)
+Accept: application/json
+X-Ngsiv2-Attrsformat: values
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+
+{
+    "data": [
```

Comment:
I have just realized about:

>  Also, ordering of attributes is not considered in this PR, that is postponed to the next one.

Thus, I understand that it will be deferred to the very next PR.

NTC